### PR TITLE
Adding SpeeDee Oil Change and Auto Services to AllThePlaces

### PR DESCRIPTION
**Pull request by tlunderwood2014**

### CODE REVIEW

File: locations/spiders/SpeeDee.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
+# -*- coding: utf-8 -*-
```
Comment (iandees) [inline locations/spiders/SpeeDee.py:1]: Please keep the file names in lowercase. Can you rename this to `speedee.py`?

**Review by tlunderwood2014 — COMMENTED** [review]
changed the file name to lowercase

**Comment by iandees** [comment]
It looks like the spider isn't returning any data. Does it work for you locally?

**Comment by tlunderwood2014** [comment]
> It looks like the spider isn't returning any data. Does it work for you locally?

Yes, It works locally for me.

**Comment by jleedev** [comment]
Cloudflare is giving a 503 for requests originating from AWS.

**Pull request merged**